nautobot/dcim/models/power.py

```diff
@@ -338,3 +339,15 @@ def get_occupied_positions(self) -> set[int]:
 
     def get_type_class(self):
         return PowerFeedTypeChoices.CSS_CLASSES.get(self.type)
+
+    @property
+    def get_utilization(self):
+        power_port = getattr(self, "connected_endpoint", None)
+        if not power_port or not hasattr(power_port, "get_power_draw"):
```

Comment:
When would `power_port` not have `get_power_draw`? Should this be replaced by something like:

```suggestion
        if not isinstance(power_port, PowerPort):
```

Reply:
Updated as requested, should be resolved. Let me know if i miss anything